syntax/std_strconv.go

```diff
@@ -0,0 +1,22 @@
+package syntax
+
+import (
+	"github.com/arr-ai/arrai/rel"
+)
+
+func stdStrconv() rel.Attr {
+	return rel.NewAttr("strconv", rel.NewTuple(
+		//TODO: eval needs to be changed to only evaluate simple expression
```

Comment:
1) Is there a use-case for splitting this functionality into two funcs? Generally the stdlib should be as simple as possible.
2) what does `unsafe` in this context mean? 
3) What is expected to happen if an "unsafe" string is passed to `eval()`?

Reply:
1 and 2. So the idea is to not give too much power to `eval`. `eval` should only be able to evaluate `Value`.  `unsafe_eval` is for the user when they need more power to evaluate expressions, but there are many problems to thing as expressions can vary wildly and it might lead to lots of problems (the example that was given is the halting problem), so it gives the idea to the user that they can use it to evaluate expression but it's up to them. As for how simple stdlib should be, there's an issue open #41 for this, might be better to discuss there.
3. As of right now, `eval()` will just evaluates it and returns a value. But, a validator will need to be implemented in the future. Right now the idea is to parse it and walk the AST and check whether it's a value or a complex expression.

Reply:
`eval` and `unsafe_eval` are roughly analogous to `eval` and `JSON.parse`, respectively, in JavaScript.

Reply:
I'm now thinking that `unsafe_eval` isn't that unsafe. JavaScript `eval` can do untold damage, including hack browser security, leak or delete user data, etc. However, because arr.ai is a hermetic expression language, `unsafe_eval` can't do anything harmful. Perhaps it makes more sense to call them `eval_value` and `eval_expr`. And yes they need to be different. See my other comment.